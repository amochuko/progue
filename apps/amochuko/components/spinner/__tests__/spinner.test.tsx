import renderer from "react-test-renderer";
import {Spinner} from "../index";

describe("Spinner Snapshot", () => {
  it("should render a large blue spinner", () => {
    const props = { loading: true, size: "large", color: "blue" };

    const comp = renderer.create(<Spinner {...props} />);
    const tree = comp.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
