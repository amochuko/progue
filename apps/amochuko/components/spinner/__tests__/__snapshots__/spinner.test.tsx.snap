// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Snapshot should render a large blue spinner 1`] = `
<div
  className="icon icon-spin text-center large blue"
/>
`;
